libopenage/gui/guisys/public/gui_renderer.h

```diff
@@ -4,11 +4,7 @@
 
 #include <memory>
 
-#ifndef __APPLE__
-#include <GL/gl.h>
-#else
-#include <OpenGL/gl.h>
-#endif
+#include <QtGui/qopengl.h>
```

Comment:
Can't really see the `QtGui/qopengl.h` in the Qt docs. It's probably more like a private header. I remember that I used `#include <QOpenGLFunctions>` somewhere instead.

The other thing is that the `gui_renderer.h` is a pimpl wrapper to isolate the openage code from Qt. So it can't include any Qt stuff. I can't try it right now, but that include should work in the `gui_renderer.cpp`.